chokonaira/Banka
# #165493175 Setup database environment

What does this PR do?
 Setup database environment

Description of Task to be completed?
Set up database environment and installed joi and pg modules
configured database schema
configured joi schema for validation
created a progress database instance and pasted URL in .env file
set up database pool connection
set up database schema table migration, reset and seed middlewares
How should this be manually tested?
clone the repo git clone git@github.com:Victor-Ugwueze/journal.git

on your terminal run cd journal

run git checkout ch-setup-database-env-165493175 

run npm install

run npm run before:start

make sure tables are migrating on the elephantQl database and admin data is seeded

Any background context you want to provide?
NA
What are the relevant pivotal tracker stories?
#165493175

Screenshots (if appropriate)
![Screenshot (164)](https://user-images.githubusercontent.com/38280567/56505745-603ef200-6514-11e9-8729-9646fed1ed38.png)
